OrbitQt/orbittablemodel.h

```diff
@@ -7,13 +7,14 @@
 #include <memory>
 #include <utility>
 
-#include "../OrbitGl/DataView.h"
+#include "DataView.h"
 
 //-----------------------------------------------------------------------------
 class OrbitTableModel : public QAbstractTableModel {
   Q_OBJECT
  public:
-  explicit OrbitTableModel(DataViewType a_Type, QObject* parent = nullptr);
+  explicit OrbitTableModel(DataView* data_view, QObject* parent = nullptr,
+                           bool alternative_row_color = true);
```

Comment:
Nit: In Qt classes it's a convention to have the parent parameter always as last.